op-devstack/presets/minimal.go

```diff
@@ -24,6 +24,8 @@ type Minimal struct {
 	L2EL      *dsl.L2ELNode
 	L2CL      *dsl.L2CLNode
 
+	Sequencer *dsl.Sequencer
```

Comment:
I think we should rename it to `TestSequencer` (both field name and stack package typing). It's not a prod service yet, and I want to avoid confusion.

Reply:
Good point. I think I renamed it everywhere, apart from the `.go` files, as I'd like to avoid messing up with tooling that detects `test` in names.